scripts/main/album.js

```diff
@@ -103,7 +103,7 @@ album.load = function(albumID, refresh = false) {
 		password: ''
 	};
 
-	processData = function(data) {
+	const processData = function (data) {
```

Comment:
Out of curiosity, what does that do?

One of these days I should really go to the trouble of learning JavaScript :wink:...

Reply:
processData is implicitly declaring a variable. Thus you should use `var`. However, for a better semantic reason instead of `var` you have `let` and `const`. 

More here:
https://stackoverflow.com/questions/40070631/v8-javascript-performance-implications-of-const-let-and-var
https://ponyfoo.com/articles/var-let-const